lib/Cargo.toml

```diff
@@ -18,7 +18,8 @@ path = "src/lib.rs"
 crate-type = ["lib"]
 
 [features]
-default = ["variable_operation"]
-#default = ["variable_operation", "minimal"]
-minimal = []
-variable_operation = []
+default = []
```

Comment:
Since not everyone might be familiar with exact way of enabling features in Cargo.toml, maybe, it is worthy to write a comment to `[features]` section how to enable features or don't remove ` #default = ["variable_operation", "minimal"]`?

Reply:
done in https://github.com/trueagi-io/hyperon-experimental/pull/495/commits/e4ede3382f812a3d9f19b950eb2511093cce33b1